pandas/_libs/src/parser/tokenizer.c

```diff
@@ -507,7 +517,7 @@ static int end_line(parser_t *self) {
             TRACE((
                 "end_line: ERROR!!! self->lines(%zu) >= self->lines_cap(%zu)\n",
                 self->lines, self->lines_cap))
-            int bufsize = 100;
+            size_t bufsize = 100;
             self->error_msg = (char *)malloc(bufsize);
             snprintf(self->error_msg, bufsize,
```

Comment:
I think doing so is a logically separate change and has some subtleties (e.g. what should allocated size be and what happens when more is needed down the road but it's not obvious someone has to check their (possibly dynamic) output length against the buffer size). Feel free to open an issue and I'll submit a patch if I have time.